project/src/main/ui/level-select/LevelSelectButton.tscn

```diff
@@ -41,13 +41,13 @@ margin_left = 4.0
 margin_top = 4.0
 margin_right = -4.0
 margin_bottom = -4.0
-size_flags_horizontal = 0
-size_flags_vertical = 0
+size_flags_vertical = 1
 custom_fonts/font = ExtResource( 6 )
 text = "Just getting started"
 align = 1
 valign = 1
 autowrap = true
+clip_text = true
```

Comment:
Maybe good to mention this `clip_text` change in the commit comment, since it was tricky.

Reply:
Expanded commit comment, `Enabled 'clip_text' property for level select buttons `